drive/sheets.py

```diff
@@ -52,9 +52,9 @@ def get_sheet_range(self, sheet_id: str, sheet_tab: str, cell_range: str,
 
     def iter_sheet_lines(self, sheet_id: str, sheet_tab: str, column_start: str, column_end: str,
                          *,
-                         offset=0,
-                         batch_size=400,
-                         sleep_for=0.5,
+                         offset: int = 0,
+                         batch_size: int = 400,
+                         sleep_for: float = 0.5,
```

Comment:
```suggestion
                         sleep_for = 0.5,
```
`0.5` is already a `float`